drivers/npmx/npmx_shell.c

```diff
@@ -485,8 +507,13 @@ static int cmd_charger_trickle_set(const struct shell *shell, size_t argc, char
 	err_code = npmx_charger_trickle_voltage_set(charger_instance, charger_trickle);
 
 	if (check_error_code(shell, err_code)) {
-		shell_print(shell, "Success: %d mV.",
-			    npmx_charger_trickle_convert_to_mv(charger_trickle));
+		uint32_t trickle_mv;
+
+		if (npmx_charger_trickle_convert_to_mv(charger_trickle, &trickle_mv)) {
```

Comment:
I'm not sure if this should be here. It's SET command. charger_trickle is provided by user.
In line 485/507 we set trickle_voltage and then we check if the value is correct?

Reply:
I agree, this should be removed, as it's not a getter.